project/src/components/main-empty/main-empty.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+function MainEmpty() {
+  return(
+    <div className="cities">
+      <div className="cities__places-container cities__places-container--empty container">
+        <section className="cities__no-places">
+          <div className="cities__status-wrapper tabs__content">
+            <b className="cities__status">No places to stay available</b>
+            <p className="cities__status-description">We could not find any property available at the moment in Dusseldorf</p>
```

Comment:
Сюда надо прокинуть город
```suggestion
            <p className="cities__status-description">We could not find any property available at the moment in {city}</p>
```